static/app/views/performance/content.spec.tsx

```diff
@@ -46,6 +45,7 @@ function initializeData(projects, query, features = FEATURES) {
     projects,
   });
   const initialData = initializeOrg({
+    ...initializeOrg(),
```

Comment:
Oh, what's this? It looks redundant

Reply:
fixing for ts with essentially defaults + override, saw this trick elsewhere and yoinked it. Free project stubs! 😆

Reply:
Interesting!